gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -1313,12 +1315,14 @@ end
 -- @realm server
 function plymeta:CanPickupWeapon(wep, forcePickup, dropBlockingWeapon)
 	self.forcedPickup = forcePickup
+	self.isPickupProbe = true
```

Comment:
As this is saved on the player it reads like the player is a Pickup Probe.
Maybe just add an argument to the hook or some other name?

Reply:
good point, for this variable I could use a hook parameter